docs/content/FormControl.mdx

```diff
@@ -0,0 +1,414 @@
+      }}
+    >
+      <FormControl>
+        <FormControl.Label>TextInputWithTokens</FormControl.Label>
+        <TextInputWithTokens onTokenRemove={onTokenRemove} tokens={tokens} />
+      </FormControl>
+      <FormControl>
+        <FormControl.Label>Autocomplete</FormControl.Label>
+        <Autocomplete>
+          <Autocomplete.Input block />
+          <Autocomplete.Overlay>
+            <Autocomplete.Menu
+              items={[
+                {text: 'css', id: 0},
+                {text: 'css-in-js', id: 1},
+                {text: 'styled-system', id: 2},
+                {text: 'javascript', id: 3},
+                {text: 'typescript', id: 4},
+                {text: 'react', id: 5},
+                {text: 'design-systems', id: 6}
+              ]}
+              selectedItemIds={[]}
+            />
+          </Autocomplete.Overlay>
+        </Autocomplete>
+      </FormControl>
+      <FormControl>
+        <FormControl.Label>Select</FormControl.Label>
+        <Select>
+          <Select.Option value="figma">Figma</Select.Option>
+          <Select.Option value="css">Primer CSS</Select.Option>
+          <Select.Option value="prc">Primer React components</Select.Option>
+          <Select.Option value="pvc">Primer ViewComponents</Select.Option>
+        </Select>
+      </FormControl>
+      <FormControl>
+        <FormControl.Label>Textarea</FormControl.Label>
+        <Textarea />
+      </FormControl>
+    </Box>
+  )
+}
+
+render(DifferentInputs)
+```
+
+### `variant="choice"` inputs
+
+```javascript live noinline
+const DifferentInputs = () => {
+  const [tokens, setTokens] = React.useState([
+    {text: 'zero', id: 0},
+    {text: 'one', id: 1},
+    {text: 'two', id: 2}
+  ])
+  const onTokenRemove = tokenId => {
+    setTokens(tokens.filter(token => token.id !== tokenId))
+  }
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      as="form"
+      sx={{
+        '> * + *': {
+          marginTop: 4
+        }
+      }}
+    >
+      <FormControl variant="choice">
+        <FormControl.Label>Checkbox option one</FormControl.Label>
+        <Checkbox />
+      </FormControl>
+      <fieldset style={{marginLeft: 0, marginRight: 0, padding: 0, border: 0}}>
+        <FormControl variant="choice">
+          <FormControl.Label>Radio option one</FormControl.Label>
+          <Radio name="radioExample" value="one" />
+        </FormControl>
+        <FormControl variant="choice">
+          <FormControl.Label>Radio option two</FormControl.Label>
+          <Radio name="radioExample" value="two" />
+        </FormControl>
+      </fieldset>
+    </Box>
+  )
+}
+
+render(DifferentInputs)
+```
+
+### Required
+
+<Note>
+  Checkbox and radio form controls cannot be required individually. Instead, you can require a selection from the entire
+  group of checkboxes or radios.
+</Note>
+
+```jsx live
+<FormControl required>
+  <FormControl.Label>Name</FormControl.Label>
+  <TextInput />
+</FormControl>
+```
```

Comment:
This note feels like it could come after the example:
````suggestion
```jsx live
<FormControl required>
  <FormControl.Label>Name</FormControl.Label>
  <TextInput />
</FormControl>
```

<Note variant="warning">
  Checkbox and radio form controls cannot be required individually. Instead, require a selection from the entire
  group of checkboxes or radios.
</Note>
````